cds-hook-services/pom.xml

```diff
@@ -358,4 +373,4 @@
 			</plugin>
 		</plugins>
 	</build>
-</project>
+</project>
```

Comment:
Another minor thing, but files should have a trailing blank line.

Reply:
done.